src/tools/rust-analyzer/crates/hir-def/src/attr.rs

```diff
@@ -81,7 +81,7 @@ impl Attrs {
                 let krate = loc.parent.lookup(db).container.krate;
                 let item_tree = loc.id.item_tree(db);
                 let variant = &item_tree[loc.id.value];
-                (variant.fields.clone(), item_tree, krate)
+                (variant.fields().clone(), item_tree, krate)
```

Comment:
Was this a find-replace mistake? Rust analyzer shouldn't be changed here lol

Reply:
ah, yes.